types/trusted-types/index.d.ts

```diff
@@ -4,30 +4,32 @@
 //                 Damien Engels <https://github.com/engelsdamien>
 //                 Emanuel Tesar <https://github.com/siegrift>
 //                 Bjarki <https://github.com/bjarkler>
+//                 Sebastian Silbermann <https://github.com/eps1lon>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.1
```

Comment:
Since this is a backwards-incompatible change, please bump to a new major version.

Reply:
That is not how versioning on DefinitelyTyped works.